src/document/DocumentAnnotator.ts

```diff
@@ -57,6 +65,18 @@ export default class DocumentAnnotator extends BaseAnnotator {
         return this.annotatedEl ? Array.from(this.annotatedEl.querySelectorAll('.page')) : [];
     }
 
+    handleChangeMode = ({ mode }: { mode: Mode }): void => {
```

Comment:
`handleModeChange`?

Reply:
I actually followed the naming convention in `BaseAnnotator` like `handleSetVisible` and `handleSetActive`